server/odc-service/src/main/java/com/oceanbase/odc/service/session/OdcStatementCallBack.java

```diff
@@ -190,39 +196,28 @@ public List<JdbcGeneralResult> doInStatement(Statement statement) throws SQLExce
             if (this.autoCommit ^ currentAutoCommit) {
                 statement.getConnection().setAutoCommit(this.autoCommit);
             }
-            Future<Void> handle = null;
-            for (SqlTuple sqlTuple : this.sqls) {
-                if (handle != null) {
-                    try {
-                        handle.get();
-                    } catch (Exception e) {
-                        // eat exception
-                    }
+            Lock lock = null;
```

Comment:
`OdcStatementCallBack` is designed for sql execution, don't add any other logic here, please

Reply:
This is the initiation of an asynchronous task executing sql, so the lock is only effective here